apps/api/src/routers/admin.py

```diff
@@ -319,6 +364,21 @@ async def _process_batch(batch: tuple[dict[str, Any], ...], decision: Decision)
     )
 
 
+async def _update_batch_of_decisions(
+    batch: tuple[dict[str, Any], ...],
+    decision: Decision,
+    *,
+    update_field: Literal["status", "decision"],
```

Comment:
Thought: I'm not sure if y'all have worked out a plan for decision release, but we previously just set the status as the final decision value without tracking a separate value for decision (see discussion in #520). If the same strategy will be used, then the `_process_batch` function could be reverted to how it was before.